Corona-Warn-App/src/main/java/de/rki/coronawarnapp/playbook/OrganizerPlaybook.kt

```diff
@@ -175,13 +174,8 @@ class OrganizerPlaybook @Inject constructor(
             ErrorType.TAN -> OrganizerSubmissionException.ErrorCode.TAN_OB_CLIENT_ERROR
             ErrorType.SUBMISSION -> OrganizerSubmissionException.ErrorCode.SUBMISSION_OB_CLIENT_ERROR
         }
-        // HTTP status code 5XX
-        is CwaServerError -> when (type) {
```

Comment:
Why is this removed?

Reply:
it is the same as `else` , it was duplicate